### ISSUE TITLE
consider moving org to `MozillaReality`

### ISSUE THREAD

**Issue by cvan** [MozillaReality/unity-webxr-export#129]
if we agree this is something we want to do, we'll want to change the links referenced from the Unity Asset Store package and settings

**Comment by cvan** [comment]
cc @larsbergstrom @edunham

**Comment by larsbergstrom** [comment]
I don't have a strong opinion, especially since @edunham already managed to get it ported over to the Mozilla GH org and permissions set up.

@edunham What are your thoughts on centralizing more of our repos into the MozillaReality GH organization vs. splitting across the two?

**Comment by edunham** [comment]
If we stick with the Mozilla org, we avoid the one-time cognitive overhead and administrative work that a move would cause for the team, and we get the branding boost of the implied direct Mozilla endorsement. In the Mozilla org it is easier for Mozilla employees to gain access without blocking on Mixed Reality team admins.

If we move only this repo to MozillaReality, we lower the barrier to entry for contributors who are already working on a project in that org but no project in the Mozilla org to be added to teams and get perms. And it'd potentially be easier for contributors to find this repo organically by looking at other repos in the MozillaReality org. Additionally moving out of the Mozilla org broadens the scope of integrations that we can safely use, since the MozillaReality org does not have highly confidential information in private repos like Mozilla does.

If we move all Mixed Reality team repos to the MozillaReality org, we would eliminate the redundancy of managing the XR team within the Mozilla org as well. 

In short, my opinion is that having all the Mixed Reality repos in the same organization makes them easier to find for contributors and avoids redundant permissions models across orgs. And bringing them together in an org other than Mozilla gives us greater flexibility for tooling integration and lessens the risk to which we expose the Mozilla org's private repos. So it's basically a good idea to bring it to the MozillaReality org eventually, but there's no big hurry and I'm happy to do the switch whenever it seems convenient to you.

**Comment by cvan** [comment]
@edunham: thanks for the thorough writeup here. this echos my thoughts - in a more coherent form :)

I agree: perhaps this repo ought to eventually live in the `MozillaReality` org (for ease of discovery) but let's continue to let this repo live in the `mozilla` org (to avoid a transfer, easy for other Mozilla members join, and the branding probably will help with legitimacy as this project matures).

let's keep the issue open but keep the repo org the same in the short term.
